include/net/golioth.h

```diff
@@ -295,6 +295,19 @@ int golioth_lightdb_set(struct golioth_client *client, const uint8_t *path,
 			enum coap_content_format format,
 			uint8_t *data, uint16_t data_len);
 
+/**
+ * @brief Delete value in Golioth's Light DB
+ *
+ * Delete value in Light DB.
+ *
+ * @param client Client instance
+ * @param path Light DB resource path
+ *
+ * @retval 0 On success
+ * @retval <0 On failure
+ */
+int golioth_lightdb_delete(struct golioth_client *client, const uint8_t *path);
```

Comment:
This doesn't match "add delete method samples" commit title. You probably need to rebase on top of main branch.